.travis.yml

```diff
@@ -75,6 +76,10 @@ after_success:
       echo "${DOCKER_API_KEY}" | docker login -u "${DOCKER_USERNAME}" --password-stdin
       docker tag "${tag}" rlworkgroup/garage-ci:latest
       docker push rlworkgroup/garage-ci
+      make build-garage-headless
+      docker push rlworkgroup/garage-headless
+      make build-garage-nvidia
```

Comment:
No no no. Do not build the nvidia dockerfile in the CI.

This won't work because the CI has no graphics card.

Also the purpose of this part of the CI script is not to maintain the Docker hub. It's to keep a cache of the CI docker image. Please remove this.

Reply:
Removed